apps/drupal/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../schema.json",
+  "name": "Drupal",
+  "port": 8215,
+  "available": true,
+  "exposable": true,
+  "id": "drupal",
+  "tipi_version": 1,
+  "version": "php8.2-fpm-bookworm",
+  "categories": [
+    "social"
```

Comment:
I don't think this category fits well drupal

Reply:
Haven has this that's why I added this...

Reply:
Haven is a social network for your family so I think it fits. I would use "development"

Reply:
Ok using that